syntax = "proto3";

message requestSequence {
  int64 sequence = 1;
}

message responseValue {
    int64 value = 1;
}

service Fibonacci {
  rpc CalcularFibo(requestSequence) returns (responseValue) {}
}
